web/client-api/types/package.json

```diff
@@ -10,9 +10,6 @@
     "url": "https://github.com/deephaven/deephaven-core.git",
     "directory": "web/client-api/types"
   },
-  "engines": {
-    "node": ">=16"
-  },
   "devDependencies": {
     "typedoc": "^0.24.8",
     "typescript": "^5.1.6"
```

Comment:
Why is this version different than the one in _proto/raw-js-openapi/package.json_?

Reply:
I'm not going to go so far as to say that I fear change, but I haven't kept up with the changes, and the "inputs" into this typescript process are generated by a tool that are no longer getting updates.